api4/config.go

```diff
@@ -285,7 +297,7 @@ func makeFilterConfigByPermission(accessType filterType) func(c *Context, struct
 				continue
 			}
 			// ConfigAccessTagWriteRestrictable trumps all other permissions
-			if tagValue == model.ConfigAccessTagWriteRestrictable {
+			if tagValue == model.ConfigAccessTagWriteRestrictable || tagValue == model.ConfigAccessTagCloudRestrictable {
```

Comment:
@jwilander Adding this condition we're also blocking writing in the `cloud_restrictable` fields similar to what we have for `write_restrictable`